AdyenActions/AdyenActionComponent.swift

```diff
@@ -192,7 +235,32 @@ public final class AdyenActionComponent: ActionComponent, ActionHandlingComponen
         
         currentActionComponent = weChatPaySDKActionComponent
     }
-    
+
+    #if canImport(TwintSDK)
```

Comment:
we can put the canImport check inside the func and do nothing if sdk isn't there

Reply:
Done 👍